### PR TITLE
[Dashboard]: Fix/3219 title and other pages UI

### PR DESCRIPTION
**Pull request by Nortsova**
## Description
- [x] All inside pages have lost their page titles on both desktop and mobile:
image

- [x]  Additional padding has been added under subtitle sections between tables across most pages.
image

- [x]  When switching between 'Teams' and 'Incoming funds' the Team page content jumps despite having the same main container width. Recording for better display 
image

The last one is not an issue; it is a scrollbar added to the teams page:

image

 ## Testing
Step 1. Check that all pages have titles now:
image
image
image
image
image

Step 2. Check paddings for Dashboard page:
Pixel perfect example:
image

Step 3. Check paddings for Members page:
Pixel perfect example:
image

Step 4. Check paddings for Income/Balance/Agreement pages: (check that padding between title and table is correct):
Pixel perfect example:
image

Step 5. Check that Dashboard mobile has correct paddings:
Pixel perfect example:
image

Additional: please check other pages and other screen sizes too. 

Contributes to #3219

### CODE REVIEW

**Review by mmioana — APPROVED** [review]
Amazing pixel perfect work @Nortsova 👏 😍 

All pages have re-gained back their title
Screenshot 2024-10-03 at 16 57 03
Screenshot 2024-10-03 at 16 57 07
Screenshot 2024-10-03 at 16 57 09
Screenshot 2024-10-03 at 16 57 12
Screenshot 2024-10-03 at 16 57 15
Screenshot 2024-10-03 at 16 57 17
Screenshot 2024-10-03 at 16 57 20
Screenshot 2024-10-03 at 16 57 24
Screenshot 2024-10-03 at 16 57 27
Screenshot 2024-10-03 at 16 57 32
Screenshot 2024-10-03 at 16 57 37
Screenshot 2024-10-03 at 16 57 40
Screenshot 2024-10-03 at 16 57 43
Screenshot 2024-10-03 at 16 57 46

Now the paddings
Screenshot 2024-10-03 at 17 01 11
Screenshot 2024-10-03 at 17 01 23
Screenshot 2024-10-03 at 17 02 51
Screenshot 2024-10-03 at 17 03 01
Screenshot 2024-10-03 at 17 03 31

And no more jumping between the pages content when switching between the pages

https://github.com/user-attachments/assets/19741038-a079-450c-909c-8c3f6926deea

Just a question @Nortsova regarding the `Agreements` page, noticed there the bottom margin between the subtitle and the table is `24px` while on the `Incoming`/`Balance` pages is `16px`. Should we address this?

Either way amaziiiing work, thanks for picking this issue up 🥇

**Comment by Nortsova** [comment]
Hey @mmioana , thank you for noticing the difference. I checked, and it seems like the Agreement page should have 24px - https://www.figma.com/design/fxgSupb31xMOYiqJSuj8gv/Agreements?node-id=401-32957&node-type=frame&t=EIAlPdx11D8o3Nqg-0

**Review by bassgeta — APPROVED** [review]
Nice work on this! I was hoping we could get rid of that pesky `PageHeadingContext`, but seeing as we need to release this soon, we can open up a new issue for that and tackle it when we're in less of a rush.
Otherwise stellar work, pixel perfect and tablet now looks like it should!
I went through the titles in a video so I don't smash a ton of images :D

https://github.com/user-attachments/assets/b661dcb9-331d-4326-93c6-10e322d3072b

The mobile dashboard is on point too :100: 
image

:ship:

File: src/components/v5/frame/ColonyHome/partials/DashboardHeader/DashboardHeader.tsx [hunk 1]
```diff
@@ -28,7 +28,7 @@ const DashboardHeader = () => {
           />
         )}
       </div>
-      <div className="flex flex-col gap-4 sm:flex-row sm:gap-7">
+      <div className="flex flex-col gap-3 sm:flex-row sm:gap-7">
```
Comment (iamsamgibbs) [inline src/components/v5/frame/ColonyHome/partials/DashboardHeader/DashboardHeader.tsx:31]: This looks like it should be gap-4 to me? Not sure if there is a design inconsistency somewhere?

Screenshot 2024-10-04 at 12 59 52

Screenshot 2024-10-04 at 13 00 00

**Review by iamsamgibbs — CHANGES_REQUESTED** [review]
Nice work on this! Page sub titles are restored!

Spacing looks really good on dashboard:

Screenshot 2024-10-04 at 12 56 40

And on members page:

Screenshot 2024-10-04 at 12 55 26

Looks like the padding isn't quite right on the Incoming, Balances or Agreements page though:

Screenshot 2024-10-04 at 12 53 00

Screenshot 2024-10-04 at 12 53 59

Figma:
Screenshot 2024-10-04 at 12 53 35

Also left a comment about the members / followers / social links spacing.

Nice job fixing up all of these tiny UI fixes though, nearly there!

**Comment by Nortsova** [comment]
Seems like this is correct padding for Agreements:
image
image


Incoming funds:
image
https://www.figma.com/design/uKxVl441Wnn02SNiOA24De/Navigation?node-id=3501-74289&node-type=frame&t=gj2k17mRSP9I5JUR-0

File: src/components/v5/frame/ColonyHome/partials/DashboardHeader/DashboardHeader.tsx (hunk 1 above)
Comment (Nortsova) [reply src/components/v5/frame/ColonyHome/partials/DashboardHeader/DashboardHeader.tsx]: reverted, thanks!

**Comment by iamsamgibbs** [comment]
The screenshot I added is from the original issue (which Mel took) I believe it from the design system: 

https://www.figma.com/design/l1dOM5qiQYwF0ElvKDqqjg/Design-System---Colony-v3?node-id=1782-28177&t=zeabG98EvLgdihOO-4

https://www.figma.com/design/l1dOM5qiQYwF0ElvKDqqjg/Design-System---Colony-v3?node-id=1782-38570&t=zeabG98EvLgdihOO-4

In this case I'm not sure whether we treat the design system (14px) as the source of truth, or the dashboard figma file (24px). Might need a decision from @melyndav

**Comment by mmioana** [comment]
> The screenshot I added is from the original issue (which Mel took) I believe it from the design system:
> 
> https://www.figma.com/design/l1dOM5qiQYwF0ElvKDqqjg/Design-System---Colony-v3?node-id=1782-28177&t=zeabG98EvLgdihOO-4
> 
> https://www.figma.com/design/l1dOM5qiQYwF0ElvKDqqjg/Design-System---Colony-v3?node-id=1782-38570&t=zeabG98EvLgdihOO-4
> 
> In this case I'm not sure whether we treat the design system (14px) as the source of truth, or the dashboard figma file (24px). Might need a decision from @melyndav

Should we better open a new issue for these paddings @iamsamgibbs @Nortsova and close this one?

**Review by iamsamgibbs — APPROVED** [review]
Thanks for this change! Let's get it merged and address the padding separately!

Screenshot 2024-10-04 at 15 17 19

**Pull request merged**